### ISSUE TITLE
namedtuple function signature

### ISSUE THREAD

**Issue by nemesmihaly** [python/mypy#16579]
**Bug Report**
The program:

from collections import namedtuple

Point = namedtuple(typename='Point', field_names=['x', 'y'])
p_1 = Point(2, 7)

The first line is marked by mypy:

Unexpected arguments to "namedtuple()"  [misc]

I think it should be allowed to use argument names in the call.

The second line is marked so:

Too many arguments for "Point"  [call-arg]

I think this is definitely a false positive.

(A clear and concise description of what the bug is.)

**To Reproduce**

```python
# Ideally, a small sample program that demonstrates the problem.
# Or even better, a reproducible playground link https://mypy-play.net/ (use the "Gist" button)
```

**Expected Behavior**

**Actual Behavior**

**Your Environment**

- Mypy version used: 1.7.1 
- Mypy command-line flags: None
- Mypy configuration options from `mypy.ini` (and other config files): None
- Python version used: 3.11.4
